### PR TITLE
Connected Django in project

### PR DESCRIPTION
**Pull request by euggeze**
Created directories, created setup.py, download Django and used command 'startproject'

### CODE REVIEW

**Review by inauros — CHANGES_REQUESTED** [review]
1. Django application must be set up using the startproject and startapp commands and then the required packages described above must be created manually. **(fixed)**
2. .pyc files should be added to .gitignore **(done)**
3. python packages must include __init__.py instead of .gitkeep (ONLY python packages, not python packages, must include .gitkeep) **(fixed)**

File: department_project/department_app/models.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (inauros) [inline department_project/department_app/models.py]: Please remove this module.

File: department_project/department_app/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (inauros) [inline department_project/department_app/tests.py]: Please remove this module.

File: department_project/department_app/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (inauros) [inline department_project/department_app/views.py]: Please remove this module.

**Review by inauros — CHANGES_REQUESTED** [review]

File: department_project/department_app/models.py (hunk 1 above)
Comment (euggeze) [reply department_project/department_app/models.py]: Done

File: department_project/department_app/tests.py (hunk 2 above)
Comment (euggeze) [reply department_project/department_app/tests.py]: Done

File: department_project/department_app/views.py (hunk 3 above)
Comment (euggeze) [reply department_project/department_app/views.py]: Done

**Review by inauros — APPROVED** [review]

**Pull request merged**